webpack.config.js

```diff
@@ -150,6 +150,51 @@ const GOOGLESITEKIT_VERSION = googleSiteKitVersion
 	? googleSiteKitVersion[ 0 ]
 	: '';
 
+/**
+ * Returns a new string with dash separators converted to
+ * camel-case equivalent.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string} string Input dash-delimited string.
+ * @return {string} Camel-cased string.
+ */
+function camelCaseDash( string ) {
+	return string.replace( /-([a-z])/g, ( match, letter ) =>
+		letter.toUpperCase()
+	);
+}
+
+const corePackages = [
+	'api-fetch',
+	'components',
+	'compose',
+	'data',
+	'dom-ready',
+	'element',
+	'hooks',
+	'icons',
+	'keycodes',
+	'scripts',
+	'url',
+];
+
+const gutenbergExternals = {
+	'@wordpress/i18n': [ 'googlesitekit', 'i18n' ],
+};
+
+corePackages.forEach( ( name ) => {
+	gutenbergExternals[ `@wordpress-core/${ name }` ] = [
+		'wp',
+		camelCaseDash( name ),
+	];
+} );
+
+const gutenbergEntryPoints = {
+	'googlesitekit-idea-hub-notice':
+		'./assets/js/googlesitekit-idea-hub-notice.js',
+};
```

Comment:
Same here. Why do we need it here? Let's move it to the config declaration to be consistent with the approach we use for other entries.